docs/topics/installing-cli-tool.adoc

```diff
@@ -24,24 +23,51 @@ Podman provides a command line interface (CLI) familiar to anyone who has used t
 For more information on installing and using Podman, see link:https://podman.io/docs/installation[Podman installation instructions].
 ====
 
+Installation options
+
+This section offers two options to install {ProductShortName} {CLINameTitle}:
+
+* Downloading a `.zip` file and installing it using a command line
```

Comment:
RedHat.TermsSuggestions - Depending on the context, consider using 'by using' or 'that uses' rather than 'using'.